citi-onboarding/Eventos-CITi
# Feat/create banner model

**### What I did**

- Create banner model at the model folder in the server
- Insert the route to get banner

**### How to test**

1. Open the terminal and go to the folder of this project
2. In terminal type:
3. git checkout feat/create-banner-model
4. cd server/
5. node install
6. node index.js
7. In your browser:
8. Open localhost http://localhost:3001/admin
9. log with e-mail: admin@gmail.com password: admin
10. Select banner
11. Try to create a banner and you should not be able to create one
12. Chick on the only model and try to edit
13. Then, save the model
14. Go to http://localhost:3001/api/banner
15. Check if error 500 is not showing up
16. Check if a Json is showed up
